app_tracker/app_tracker/models.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.models import User
+from django.db import models
+from djmoney.models.fields import MoneyField
+
+class Application(models.Model):
+    company = models.CharField(null=False, max_length=64)
+    date_applied = models.DateField(null=True)
+    first_interview = models.BooleanField(default=False)
+    hiring_manager = models.CharField(null=True, blank=False, max_length=64)
+    job_description = models.CharField(null=False, unique=False, max_length=64, default=None)
+    notes = models.CharField(null=True, blank=False, max_length=250)
+    offer_extended = models.BooleanField(default=False)
+    recruiter = models.CharField(null=True, blank=False, max_length=64)
+    second_interview = models.BooleanField(default=False)
+    user = models.ForeignKey(User, related_name="applications", on_delete=models.PROTECT, null=False)
+
+class Offer(models.Model):
+    class OffersTypeChoices(models.TextChoices):
+        CONTRACT = "Contract"
+        CONTRACT_TO_HIRE = "Contract to Hire"
+        FULL_TIME = "Full Time"
+        PART_TIME = "Part Time"
+        
+    application = models.ForeignKey(
+        Application, related_name="offers", null=True, on_delete=models.PROTECT
+    )
+    date_offered = models.DateField(null=False)
+    dental = models.BooleanField(default=False)
+    healthcare = models.BooleanField(default=False)
+    length = models.CharField(null=True, blank=False, max_length=64)
+    notes = models.CharField(null=True, blank=False, max_length=250)
+    pto = models.CharField(null=True, blank=False, max_length=64)
+    retirement = models.BooleanField(default=False)
+    salary = MoneyField(
+        max_digits=14, decimal_places=2, default_currency="USD", null=True
+    )
+    type = models.CharField(
+        null=True, blank=False, default=OffersTypeChoices.FULL_TIME, choices=OffersTypeChoices.choices, max_length=32
+    )
+    user = models.ForeignKey(User, related_name="offers", on_delete=models.PROTECT, null=False)
+    vision = models.BooleanField(default=False)
+
+class Todo(models.Model):
+    completed = models.BooleanField(default=False)
+    date_created = models.DateField(null=False)
+    name = models.CharField(null=False, blank=False, unique=True, max_length=64)
+    user = models.ForeignKey(User, related_name="todos", on_delete=models.PROTECT, null=False)
```

Comment:
Missing new line. 

Also you will have enough information on recuiters and hiring managers that you should add models for them. Hiring manager can have name and email. Recruiter can have name, company, email, and phone number. Use the [django-phone-number-field](https://github.com/stefanfoulis/django-phonenumber-field) library for this.

Reply:
Created models for `Company`, `HiringManager`, and `Recruiter`. Used the above phone number field.